### PR TITLE
🔍 Tidying up

### LINKED ISSUE #353: 🕵️‍♀️ aligning the search box/mag glass
Screen Shot 2019-09-12 at 12 04 15 PM

### PR DESCRIPTION
**Pull request by mermop**
Closes #353 and #379 

- Refactors a bunch of CSS for general consistency
- Adds labels to the form elements - previously they only had placeholders, which are invisible once you've entered text
- Moves the magnifying glass to its own search button
- Some CSS refactoring to make defaults less specific
- Visual design feedback from Paul

Screen Shot 2019-09-17 at 3 39 19 PM

### CODE REVIEW

**Comment by JacOng17** [comment]
border
Maybe can add a border around the categories so that's its easier to select/identify?

File: src/assets/scss/components/_category.scss [hunk 1]
```diff
@@ -28,23 +28,11 @@
...
-    color: mix(#000, #FFF, 50);
-    text-align: right;
-    width: 100%;
-  }
 }
 
-.green-expansion-panel {
-  border: 1px solid $primary-color;
+.category-expansion-panel {
+  border: 1px solid #eee;
```
Comment (JacOng17) [inline src/assets/scss/components/_category.scss]: Should #eee be a color variable instead?

File: src/assets/scss/components/_search.scss [hunk 2]
```diff
@@ -6,18 +6,14 @@
   }
 
   &__container {
-    padding-bottom: 1rem;
-    margin-bottom: 1rem;
-    max-width: 40rem;
-
     button {
       font-size: 1rem;
       margin: .25rem .5rem .25rem 0;
```
Comment (JacOng17) [inline src/assets/scss/components/_search.scss]:
```suggestion
      margin: .25rem .5rem .25rem 0;
      border: 1px solid black;
```

Comment (JacOng17) [reply src/assets/scss/components/_search.scss]: Attached the screenshot above.

**Comment by Br3nda** [comment]
feedback: 
the green is very green
the category selector doesn't feel visually linked and repeats itself
less eeeeeeeee, more aaaaaaaa

**Comment by Br3nda** [comment]
task from Paul:
- [ ] make the categories square

**Comment by Br3nda** [comment]
@JacOng17 we changed the colours. check out the pretty!

File: src/assets/scss/components/_search.scss (hunk 2 above)
Comment (mermop) [reply src/assets/scss/components/_search.scss]: Just changed this to pink on feedback from the client :)

File: src/assets/scss/components/_category.scss (hunk 1 above)
Comment (mermop) [reply src/assets/scss/components/_category.scss]: Good point - have changed

File: src/assets/scss/components/_search.scss [hunk 3]
```diff
@@ -32,3 +28,12 @@
     text-indent: -9999px;
   }
 }
+
+.react-autosuggest__label, .small-label {
+  font-size: 0.9em;
+  margin-bottom: 0.2em;
```
Comment (JacOng17) [inline src/assets/scss/components/_search.scss]:
```suggestion
   font-weight: bold;
   margin-bottom: 0.2em;
```

Comment (JacOng17) [reply src/assets/scss/components/_search.scss]: Just so the labels are more defined.

**Pull request merged**